### PR TITLE
feat(registry): add support for setting private registry information per application

### PR DESCRIPTION
**Pull request by helgi**
ref https://github.com/deis/controller/pull/653

### CODE REVIEW

**Comment by rimusz** [comment]
👍

**Comment by bacongobbler** [comment]
Looks like this is a direct copy of `config`. Why not just make it into a [shortcut](https://github.com/deis/workflow-cli/blob/6c176d28c6dc5b8da290e119ecadab744a5645a8/deis.go#L199) instead?

File: cmd/registry.go [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+		return err
+	}
+
+	config, err := config.List(c, appID)
+
+	fmt.Printf("=== %s Registry\n", appID)
+
+	registryMap := make(map[string]string)
+
+	for key, value := range config.Registry {
```
Comment (bacongobbler) [inline cmd/registry.go]: Ah I see here why. Makes sense.

Comment (helgi) [inline cmd/registry.go]: Same reason as tags, I wanted to do a short cut tho - maybe we can unify them all later?

**Pull request merged**